addons/l10n_jp/data/account.account.template.csv

```diff
@@ -1,124 +1,129 @@
-B72201,B72201,投資有価証券売却損(特損),expense,,FALSE,,l10n_jp1
-B72202,B72202,投資有価証券評価損(特損),expense,,FALSE,,l10n_jp1
-B80101,B80101,法人税、住民税及び事業税,expense,,FALSE,,l10n_jp1
-B90001,B90001,期間利益,expense,,FALSE,,l10n_jp1
+id,chart_template_id/id,code,account_type,name,reconcile,currency_id/id,tag_ids/id
+l10n_jp_110000,l10n_jp_chart_template,110000,asset_fixed,Buildings and Structures,FALSE,,
+l10n_jp_110100,l10n_jp_chart_template,110100,asset_fixed,Buildings and Structures - Accumulated Depreciation,FALSE,,
+l10n_jp_110200,l10n_jp_chart_template,110200,asset_fixed,Machinery and Equipment,FALSE,,
+l10n_jp_110300,l10n_jp_chart_template,110300,asset_fixed,Machinery and Equipment - Accumulated Depreciation,FALSE,,
+l10n_jp_110400,l10n_jp_chart_template,110400,asset_fixed,"Tools, Furniture and Fixtures",FALSE,,
+l10n_jp_110500,l10n_jp_chart_template,110500,asset_fixed,"Tools, Furniture and Fixtures - Accumulated Depreciation",FALSE,,
+l10n_jp_110600,l10n_jp_chart_template,110600,asset_fixed,Land,FALSE,,
+l10n_jp_120003,l10n_jp_chart_template,120003,asset_cash,Deposit,FALSE,,
+l10n_jp_120004,l10n_jp_chart_template,120004,asset_cash,Time Deposit,FALSE,,
+l10n_jp_121000,l10n_jp_chart_template,121000,asset_current,Current Assets,FALSE,,
+l10n_jp_121100,l10n_jp_chart_template,121100,asset_current,Stock Valuation Account,FALSE,,
+l10n_jp_121200,l10n_jp_chart_template,121200,asset_current,Stock Interim Account (Received),FALSE,,
+l10n_jp_121300,l10n_jp_chart_template,121300,asset_current,Stock Interim Account (Delivered),FALSE,,
+l10n_jp_122000,l10n_jp_chart_template,122000,asset_current,Merchandise,FALSE,,
+l10n_jp_122100,l10n_jp_chart_template,122100,asset_current,Finished Goods,FALSE,,
+l10n_jp_122200,l10n_jp_chart_template,122200,asset_current,Semi-finished Goods,FALSE,,
+l10n_jp_122300,l10n_jp_chart_template,122300,asset_current,Inventory in Transit,FALSE,,
+l10n_jp_122400,l10n_jp_chart_template,122400,asset_current,Work in Process,FALSE,,
+l10n_jp_122500,l10n_jp_chart_template,122500,asset_current,Raw Materials,FALSE,,
+l10n_jp_122600,l10n_jp_chart_template,122600,asset_current,Materials,FALSE,,
+l10n_jp_122700,l10n_jp_chart_template,122700,asset_current,Advance Payments,FALSE,,
+l10n_jp_122800,l10n_jp_chart_template,122800,asset_current,Prepaid Expenses,FALSE,,
+l10n_jp_122900,l10n_jp_chart_template,122900,asset_current,Advances Paid,FALSE,,
+l10n_jp_123000,l10n_jp_chart_template,123000,asset_current,Suspense Payments,FALSE,,
+l10n_jp_123100,l10n_jp_chart_template,123100,asset_current,Lease/Guarantee Deposits,FALSE,,
+l10n_jp_123200,l10n_jp_chart_template,123200,asset_current,Consumption Tax - Purchase,FALSE,,
+l10n_jp_123300,l10n_jp_chart_template,123300,asset_current,Bad Debt Allowance,FALSE,,
+l10n_jp_123400,l10n_jp_chart_template,123400,asset_current,Accrued Income,FALSE,,
+l10n_jp_123500,l10n_jp_chart_template,123500,asset_current,Securities,FALSE,,
+l10n_jp_chart_template_liquidity_transfer,l10n_jp_chart_template,123600,asset_current,Liquidity Transfer,TRUE,,
```

Comment:
I need to use this name such that in https://github.com/odoo/odoo/blob/16.0/addons/account/models/chart_template.py#L859 it will found this account and use this as the liquidity transfer.